src/pages/userManagement/UserManagerCollaborationEditTable.jsx

```diff
@@ -39,14 +34,10 @@ export default function UserManagersCollaborationEditTable({
     mutate: restoreCollab,
     success: restoreSuccess,
     clearSuccess: clearRestoreSuccess,
-    loading: restoreLoading,
     error: restoreError,
     clearError: onClearRestoreError,
   } = usePatchCollaboration();
 
-  const isLoading =
-    userDataLoading || revokeLoading || restoreLoading;
```

Comment:
currently, this table doesn't use a loading state or update the data after revoke or restore actions are clicked. eventually these loading statuses will need to be used but right now they were just causing a grey box to appear below the table.

Reply:
That's not the behavior that I see. Revoking and restoring do update the table. While they are in progress, the entire table shows a progress bar. `collaborationLoading` doesn't change during revoke or restore, so the skeleton only shows up on the initial page load.

Reply:
This is also not the behavior that I see:
<img width="1840" alt="Screen Shot 2022-06-22 at 2 25 18 PM" src="https://user-images.githubusercontent.com/2775448/175141361-3c5c5897-48b1-452c-959b-fad73f66409e.png">

<img width="1840" alt="Screen Shot 2022-06-22 at 2 25 26 PM" src="https://user-images.githubusercontent.com/2775448/175141324-9efa3a77-4e60-4dd9-89da-8f0888f1788b.png">
.